src/lib/command_runner/commands_components/NewWithTemplate.svelte

```diff
@@ -11,8 +11,6 @@
 				class="action-confirm"
 				title="Open {template}"
 				href="./?t={template}"
-				target="_blank"
```

Comment:
Ah okay we shall reload afterwards then. Opening up in a new tab is annoying for installed pwa and is an a11y concern since you take agency away but it shall be another pr

Reply:
If you can commit the change and merge id be glad I'm out right now